test/Microsoft.AspNet.Localization.Tests/AccessLanguageHeaderRequestCultureProviderTest.cs

```diff
@@ -106,5 +106,38 @@ public class AccessLanguageHeaderRequestCultureProviderTest
                 var response = await client.GetAsync(string.Empty);
             }
         }
+        
+        [Fact]
+        public async void OmitDefaultRequestCultureShouldNotThrowNullReferenceException_And_ShouldGetTheRightCulture()
```

Comment:
The line you pointed to is `GetFallbackLanguage_ReturnsFromSupportedCulture_AcceptLanguageListContainsSupportedCultures`. I am talking about `GetFallbackLanguage_ReturnsFirstNonNullCultureFromSupportedCultureList`